optimal_control_python/up_and_down_bow.py

```diff
@@ -54,6 +72,9 @@ def neck_marker(self):
             }[self.string]
 
     def rt(self):
+        """
+        :return: RT number according to the string.
```

Comment:
RT that describes the orientation of bow RT that would properly play on that string
(or something like that..)